libraries/classes/Util.php

```diff
@@ -2860,6 +2860,7 @@ public static function printableBitValue(int $value, int $length): string
      */
     public static function convertBitDefaultValue($bit_default_value)
     {
+        if ($bit_default_value === NULL) $bit_default_value = "";
         return preg_replace("/^b'(\d*)'?$/", '$1', htmlspecialchars_decode($bit_default_value, ENT_QUOTES), 1);
```

Comment:
```suggestion
        return preg_replace("/^b'(\d*)'?$/", '$1', htmlspecialchars_decode((string) $bit_default_value, ENT_QUOTES), 1);
```
Would be the best fix

Reply:
Yes :-)